quantumdb-cli/src/main/java/io/quantumdb/cli/xml/ChangelogLoader.java

```diff
@@ -16,6 +17,26 @@ public Changelog load(Changelog changelog, String file) throws IOException {
 		XmlChangelog xml = new XmlMapper().loadChangelog(file);
 		List<XmlChangeset> changesets = xml.getChangesets();
 
+		HashSet<String> idSet = new HashSet<>();
+		Version temp = changelog.getRoot();
+		idSet.add(temp.getId());
+		while (temp.getChild() != null) {
+			temp = temp.getChild();
+			if (idSet.contains(temp.getId())) {
+				throw new IllegalStateException("Existing changelog in the database already has two id's with the name: " + temp.getId() + ", please change one id in the database.");
+			}
+			idSet.add(temp.getId());
+		}
+
+		for (XmlChangeset changeset : changesets) {
+			if (idSet.contains(changeset.getId())) {
+				throw new IllegalStateException("Changelog must contain only unique id's, id: " + changeset.getId() + " is already present.");
+			}
+			else {
+				idSet.add(changeset.getId());
+			}
+		}
+
```

Comment:
Throw error when multiple id's with the same name are present as right now the id is used to perform the migrations.